.solcover.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  skipFiles: ['mocks', 'interfaces'],
+  skipFiles: ['mocks', 'interfaces', 'test', 'periphery/DepositPeriphery.sol'],
```

Comment:
Are we sure we do not want to include DepositPeriphery in coverage?

Reply:
yes, we can also remove `test` folder